packages/sdk/src/mcp/fs/api.ts

```diff
@@ -242,6 +256,17 @@ export const deleteFile = createTool({
       Key: path,
     });
 
+    const { error } = await c.db.from("deco_chat_assets").delete().eq(
+      "workspace",
+      c.workspace.value,
+    ).eq("file_url", path).select().single();
+
+    if (error) {
+      console.log(
+        `failed deleting file: ${path} - workspace: ${c.workspace.value}`,
+      );
+    }
```

Comment:
shouldn't throw an error?

Reply:
I mean, you should distinguish between the entity does not exists and a valid error maybe?